packages/uswds-core/src/styles/mixins/general/table.scss

```diff
@@ -5,60 +5,67 @@
 @use "../utilities" as *;
 @use "../typography/typeset" as *;
 
-$table-text-color: color(
-  get-color-token-from-bg(
-    $bg-color: $theme-body-background-color,
-    $preferred-text-token: $theme-text-reverse-color,
-    $fallback-text-token: $theme-table-text-color,
-    $context: "Table text",
-  )
+$table-text-color: get-color-token-from-bg(
+  $bg-color: $theme-body-background-color,
+  $preferred-text-token: $theme-table-text-color,
+  $context: "Table text",
 );
 
 $table-header-text-color: color(
   get-color-token-from-bg(
     $bg-color: $theme-table-header-background-color,
-    $preferred-text-token: $theme-text-reverse-color,
-    $fallback-text-token: $theme-table-header-text-color,
```

Comment:
Moved settings into the value for `$preferred-text-token`. Allowed `$fallback-text-token` to use the fallback from project defaults.